src/main/resources/application.properties

```diff
@@ -0,0 +1,17 @@
+
+profile.active=prod
```

Comment:
Do you use Spring profiles?

Reply:
same, let's avoid application behaviours/configurations based on the environment, leave the deployment decide the correct configurations and be agnostic in the service